packages/asc-ui/src/components/Button/ButtonStyle.ts

```diff
@@ -2,7 +2,6 @@ import { darken, readableColor, size, transitions } from 'polished'
 import styled, { css } from 'styled-components'
```

Comment:
There are still two instances in this file where the `outline` is set which might interfere with the browser styling.

Reply:
only for hover and disabled state. shouldn't interfere I guess, right?

Reply:
Yeah, should be ok. Just double checking.